constantinCurtis/src/main/java/data/collection/task/classes/Address.java

```diff
@@ -22,6 +22,30 @@ public Address(String country, String city, String street) {
         this.street = street;
     }
 
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getStreet() {
```

Comment:
you are never using this method, still we need it?

Reply:
deleted